dcos_installer/cli.py

```diff
@@ -136,7 +136,7 @@ def do_validate_config(args):
     'genconf': (
         lambda args: backend.do_configure(),
         'EXECUTING CONFIGURATION GENERATION',
-        'Execute the configuration generation (genconf).'),
+        'Create customized DC/OS build files for the cluster in {}.'.format(dcos_installer.constants.SERVE_DIR)),
```

Comment:
I think the serve dir location is more detail than users would need. If you're using the CLI install method then you don't need to know anything about the contents of `genconf/serve/` since it copies install files across the cluster for you, and the advanced install doc tells you which install files you need to host on the bootstrap node. The only file under `genconf/` a user absolutely must know about to install DC/OS onprem is `config.yaml`.

I'd propose something like:

```
'Create DC/OS install files customized according to {}.'.format(dcos_installer.constants.CONFIG_PATH)
```